helm/tailing-sidecar-operator/values.yaml

```diff
@@ -18,3 +18,6 @@ sidecar:
   image:
     repository: public.ecr.aws/sumologic/tailing-sidecar
     tag: 0.0.0
+
+
+useCertManager: false
```

Comment:
I would go with some `enabled` option

Reply:
I think that it will be good to have additional key - `certManager` if in the future there will be a need to add more options for cert-manger.

Reply:
> I think that it will be good to have additional key - `certManager` if in the future there will be a need to add more options for cert-manger.

Right, I agree this makes sense. My only gripe is `certManager.enabled` looks to me a bit like the other flags where we enable a chart dependency, where this flag does not enable a chart dependency but only a feature of the chart.